### PR TITLE
update base.mcf for Provenance/Dataset/Source

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

File: deploy/mapping/base.mcf [hunk 1]
```diff
@@ -170,6 +170,7 @@ description: C:Provenance->provenance_description
 importUrl: C:Provenance->mcf_url
 importTime: C:Provenance->timestamp_secs
 importDuration: C:Provenance->duration_secs
+isPartOf: C:Provenance->is_part_of
```
Comment (pradh) [inline deploy/mapping/base.mcf:173]: I think this should be `E:Provenance->E5`, where E5 is:

```
Node: E:Provenance->E5
typeOf: Dataset
dcid: C:Provenance->is_part_of
functionalDeps: dcis
```

File: deploy/mapping/base.mcf [hunk 2]
```diff
@@ -208,13 +209,28 @@ typeOf: Source
...
 dcid: C:Source->prov_id
 functionalDeps: dcid
 
+Node: E:Dataset->E1
+typeOf: Dataset
+dcid: C:Dataset->id
+provenance: E:Dataset->E2
+name: C:Dataset->name
+url: C:Dataset->url
+isPartOf: C:Dataset->is_part_of
```
Comment (pradh) [inline deploy/mapping/base.mcf:227]: Like above this should be E3 where E3 is a Source.

File: deploy/mapping/base.mcf (hunk 1 above)
Comment (pradh) [reply deploy/mapping/base.mcf:173]: Sorry I missed this in the other mapping.

Comment (n-h-diaz) [reply deploy/mapping/base.mcf]: got it thanks! will update

File: deploy/mapping/base.mcf (hunk 2 above)
Comment (n-h-diaz) [reply deploy/mapping/base.mcf]: done

File: deploy/mapping/base.mcf [hunk 3]
```diff
@@ -188,6 +188,11 @@ typeOf: Provenance
 dcid: C:Provenance->prov_id
 functionalDeps: dcid
 
+Node: E:Provenance->E5
```
Comment (pradh) [inline deploy/mapping/base.mcf:191]: We need an `isPartOf: E:Provenance->E5` added to the E:Provenance->E1 node?

File: deploy/mapping/base.mcf [hunk 4]
```diff
@@ -216,6 +223,25 @@ dcid: C:Source->prov_id
 functionalDeps: dcid
 
 
+Node: E:Dataset->E1
+typeOf: Dataset
+dcid: C:Dataset->id
+provenance: E:Dataset->E2
+name: C:Dataset->name
+url: C:Dataset->url
```
Comment (pradh) [inline deploy/mapping/base.mcf:231]: ditto.  add `isPartOf` here?

Comment (n-h-diaz) [reply deploy/mapping/base.mcf:232]: done

**Comment by n-h-diaz** [comment]
> Thanks!
> 
> Lets just sanity check this with @shifucun for a sec before merging :)

ahhh sorry I didn't see this - I can revert the PR if there are any concerns with this

**Pull request merged**